pages/common/routers/form.js

```diff
@@ -18,17 +18,32 @@ const flattedNested = (data, schema) =>
     : data[key]
   );
 
+const persistValues = req => {
+  merge(req.session.form[req.model.id], {
+    values: pickBy(req.form.values, (field, key) => {
+      return req.form.schema[key].persist !== false;
+    })
+  });
+};
+
```

Comment:
Does anything bad happen if we persist all the values?

Reply:
nope!